Fabric.Authorization.AccessControl/src/app/services/interceptors/fabric-http-error-handler-interceptor.service.ts

```diff
@@ -33,6 +37,8 @@ export class FabricHttpErrorHandlerInterceptorService {
               response.error
             )}`
           );
+
+          this.showAlert(response, response.statusText);
```

Comment:
Is the response.statusText informative enough? Not sure what the alternative would be though.